include/game.h

```diff
@@ -235,6 +240,10 @@ extern struct Camera gCamera;
 #define IS_OUT_OF_CAM_RANGE_TYPED(castType, x, y)                                       \
     IS_OUT_OF_RANGE(castType, x, y, CAM_REGION_WIDTH)
 
+#define IS_OUT_OF_GRAV_TRIGGER_RANGE(x, y)                                              \
+    (((u16)(x + (CAM_REGION_WIDTH / 2)) > CAM_BOUND_X) || (y < -(CAM_REGION_WIDTH / 2)) \
```

Comment:
These casts can be represented as range check statements `x > ... && x < ...` ? I'm not exactly sure how but it will be possible at some point